### PR TITLE
LPS-137535 Update custom object field integration test

### PR DESCRIPTION
**Pull request by gabrielwas**

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 1]
```diff
@@ -81,9 +81,10 @@ public ObjectDefinition postObjectDefinition(
 
 		return _toObjectDefinition(
 			_objectDefinitionService.addCustomObjectDefinition(
-				LocalizedMapUtil.getLocalizedMap(objectDefinition.getName()),
+				LocalizedMapUtil.getLocalizedMap(objectDefinition.getLabel()),
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:84]: Hey @gabrielwas , these fields have already been created in the UI? I passed label and pluralLabel with name value to not break the UI while Adriano is working on that.

Comment (gabrielwas) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:84]: @guilhermedcamacho I'm filling the labelMap if it is empty in the service layer.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java [hunk 2]
```diff
@@ -527,6 +529,14 @@ private ObjectDefinition _addObjectDefinition(
...
 		}
 
+		if (MapUtil.isEmpty(labelMap)) {
+			labelMap = LocalizedMapUtil.getLocalizedMap(name);
+		}
+
+		if (MapUtil.isEmpty(pluralLabelMap)) {
+			pluralLabelMap = LocalizedMapUtil.getLocalizedMap(name);
+		}
+
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java:539]: @guilhermedcamacho here

**Comment by gabrielwas** [comment]
ci:stop

**Comment by guilhermedcamacho** [comment]
ci:close

**Pull request closed without merging**